vendor/cookbooks/rails/recipes/databases.rb

```diff
@@ -1,30 +1,42 @@
-include_recipe "database"
-
 if node[:active_applications]
   node[:active_applications].each do |app, app_info|
     if app_info['database_info']
       database_info = app_info['database_info']
-      database_name = app_info['database_info']['database']
+      
+      database_name     = database_info['database']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.